AttitudeManager/Src/AM.cpp

```diff
@@ -50,15 +50,14 @@ std::vector<ActuatorOutput> AttitudeManager::runTransitionMixingIteration(
     std::vector<ActuatorOutput> desired_output =
         desired_controller->runControlsAlgorithm(instructions);
 
-#warning This should be getting the current time
+//#warning This should be getting the current time
     // Determine how much of each mix to apply
-    const uint64_t current_time_ms = 0;  // TODO: Get the current time from los
+    const uint64_t current_airspeed = 0; // get from somewhere? path man
     const float transition_percent =
-        (transition_start_time_ms - current_time_ms) / transition_time_ms;
+        (current_airspeed - transition_start_airspeed) / desired_airspeed;
```

Comment:
What are you trying to calculate? I don't think this works the way you were intending?